acme/dns_challenge.go

```diff
@@ -69,7 +68,7 @@ func (s *dnsChallenge) Solve(chlng challenge, domain string) error {
 
 	logf("[INFO][%s] Checking DNS record propagation...", domain)
 
-	err = waitFor(30, 2, func() (bool, error) {
+	err = WaitFor(30, 2, func() (bool, error) {
```

Comment:
Can we bump this up to 60 seconds while you are at it? The current value has proven to be a bit too optimistic.